cpp/src/plasma/malloc.cc

```diff
@@ -40,7 +42,7 @@ int fake_munmap(void*, int64_t);
 #define USE_DL_PREFIX
 #define HAVE_MORECORE 0
 #define DEFAULT_MMAP_THRESHOLD MAX_SIZE_T
-#define DEFAULT_GRANULARITY ((size_t)128U * 1024U)
+#define DEFAULT_GRANULARITY ((size_t) 1024U * 1024U * 1024U) // 1GB
```

Comment:
in fact, one improvement I've been thinking about is exponential backoff on default granularity + mmap retry on mmap failure. As files we attempt to mmap double in size, we can easily get stuck with only 50% memory capacity mmaped and unable to mmap the remaining 50% (due to ENOMEM).